docs/README.md

```diff
@@ -69,6 +69,7 @@ Per group of metrics there is one file for each metrics. See each file for speci
 - [Ingress Metrics](ingress-metrics.md)
 - [CertificateSigningRequest Metrics](certificatessigningrequest-metrics.md)
 - [VerticalPodAutoscaler Metrics](verticalpodautoscaler-metrics.md)
+- [NetworkPolicy Metrics](networkpolicy-metrics.md)
```

Comment:
Something we haven't been enforcing well enough - lexicographical order. Let's place this above Node metrics.

Reply:
Would you like me to just sort the whole list alphabetically?

Reply:
That would be great actually @milesbxf :)